src/wallet/components/ConnectWallet.tsx

```diff
@@ -72,23 +84,22 @@ export function ConnectWallet({
   }, [status, hasClickedConnect, onConnect]);
 
   if (status === 'disconnected') {
-    if (withWalletAggregator) {
+    if (config?.wallet?.display === 'modal') {
       return (
-        <ConnectButtonRainbowKit.Custom>
-          {({ openConnectModal }) => (
-            <div className="flex" data-testid="ockConnectWallet_Container">
-              <ConnectButton
-                className={className}
-                connectWalletText={connectWalletText}
-                onClick={() => {
-                  openConnectModal();
-                  setHasClickedConnect(true);
-                }}
-                text={text}
-              />
-            </div>
+        <div className="flex" data-testid="ockConnectWallet_Container">
+          <ConnectButton
+            className={className}
+            connectWalletText={connectWalletText}
+            onClick={() => {
+              handleOpenModal();
+              setHasClickedConnect(true);
```

Comment:
Another way to do this (and to remove the state for `hasClickedConnect` would be to just set if the modal _was_ open and the `status === 'connected'`, but I think it's fine to leave for now. 

Previously we didn't have full control over the modal with Rainbowkit